doc/02-Installation.md

```diff
@@ -198,10 +218,14 @@ systemctl start icingadb
 
 ### Enable remote Redis connections <a id="remote-redis"></a>
 
-By default `icingadb-redis` listens only on `127.0.0.1`. If you want to change that (e.g. Icinga Web 2), just change `bind 127.0.0.1 ::1` and `protected-mode yes` in `/etc/icinga-redis/icingadb-redis.conf` to the interface you want to use and `protected-mode no`.
+By default `icingadb-redis` listens only on `127.0.0.1`. If you want to change that (e.g. Icinga Web 2), just
+change `bind 127.0.0.1 ::1` and `protected-mode yes` in `/etc/icingadb-redis/icingadb-redis.conf` to the interface you
+want to use and `protected-mode no`.
 
-> WARNING: Make sure your host is secured by some kind of firewall, if you open Redis to an external interface. Redis, by default, does not have any authentication that prevent others from accessing it.
+> WARNING: Make sure your host is secured by some kind of firewall, if you open Redis to an external interface.
+> Redis, by default, does not have any authentication that prevent others from accessing it.
```

Comment:
We should probably mention TLS and the possibility to set a password or use client certificates here.